File: Modul 5/soal 3/PRAK403-2210817310006-Ahmad Muhsin Zein.py
angka1, angka2 = input().split()
angka1 = int(angka1)
angka2 = int(angka2)
if angka1 < angka2:
    for i in range(angka1, angka2):
        print(angka1, angka2, end=' - ')
        angka1 += 1
        angka2 -= 1
        continue
    print(angka1, angka2, end='  ')
    angka1 += 1
    angka2 -= 1
else:
    if angka1 > angka2:
        n = angka1
        n1 = angka2
        for i in range(angka2, angka1):
            print(angka1, angka2, end= ' - ')
            angka1 -= 1
            angka2 += 1
            if angka1 % n1 == 1 :
                break
    if angka1 < angka2:
        for i in range(angka1, angka2):
            print(angka1, angka2, end=' - ')
            angka1 -= 1
            angka2 += 1
            break
        print(angka1, angka2, end= ' ')
        angka1 -= 1
        angka2 += 1

File: Modul 5/soal 3/PRAK403-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

int main() {
    int angka1, angka2, i, j, x, y;
    scanf("%d %d", &angka1, &angka2);
    x = angka1;
    y = angka2;

    if (angka1 > angka2) {
        for(i = angka1, j = angka2; i >= angka2, j <= angka1; i--, j++) {
                while(angka1 >= angka2){
                printf("%d %d - ", angka1, angka2);
                angka1--;
                angka2++;
                continue;
            }
            printf("%d %d - ", angka1, angka2);
            angka1--;
            angka2++;
            if(angka1 == y){
                    printf("%d %d", angka1, angka2);
            }
        }
    }
    else {
        for(i = angka1, j = angka2; i <= angka2, j >= angka1; i++, j--) {
                while(angka1 <= angka2){
                printf("%d %d - ", angka1, angka2);
                angka1++;
                angka2--;
                continue;
            }
            printf("%d %d - ", angka1, angka2);
            angka1++;
            angka2--;
            if(angka2 == x){
                    printf("%d %d", angka1, angka2);
            }
        }
    }
    return 0;
}
